libraries/adaptive-expressions/package.json

```diff
@@ -21,27 +21,9 @@
   "typings": "./lib/index.d.ts",
   "dependencies": {
     "@microsoft/recognizers-text-data-types-timex-expression": "1.3.0",
-    "@types/atob-lite": "^2.0.0",
```

Comment:
There's no way this is allowed, the build will fail. We use these dependencies all over the `adaptive-expressions` codebase.